src/left-nav.jsx

```diff
@@ -41,9 +43,11 @@ const LeftNav = React.createClass({
     className: React.PropTypes.string,
     disableSwipeToOpen: React.PropTypes.bool,
     docked: React.PropTypes.bool,
+    open: React.PropTypes.bool,
```

Comment:
yes sorry, I have just discovered this later. I changed to `null` as here https://github.com/callemall/material-ui/blob/master/src/dialog.jsx#L133